### DISCUSSION TITLE
Invitation code on invite page

### DISCUSSION THREAD

**Discussion by aftechro** [Ylianst/MeshCentral#4316] [Q&A] [answered]
Hello,

Can anyone advise me on where can i get the invite code for the https://mesh-domain.com/invite that i need to share with the client.

<img width="391" alt="image" src="https://user-images.githubusercontent.com/38830718/180652482-3443f5d9-e2d4-4b08-967f-4fcba95e430c.png">

i was looking on the internet for answers but couldn't find anything related.

Thank you!

**Comment by Ylianst** [comment, marked as answer]
Sorry for the delay. Click on a device group and look for the "Invite Codes" line. It should look like this:

![image](https://user-images.githubusercontent.com/1319013/182712864-24d2f0ad-f1c0-4c5f-8aca-f53836b0223a.png)

You can then use the dialog box to set one or more invitation codes for agent installation. You probably have this already, but to activate this feature to need to following line in the "domains" section of the config.json:

```
"agentInviteCodes": true
```

Let me know if that works.

**Comment by aftechro** [comment]
that`s great, thank you!
